test/e2e/models/CodeServer.ts

```diff
@@ -78,6 +78,7 @@ export class CodeServer {
         "node",
         [
           process.env.CODE_SERVER_TEST_ENTRY || ".",
+          ...this.codeServerArgs,
```

Comment:
✍🏼 We don't want to accidentally override CLI flags expected by our tests suites so pass `this.codeServerArgs` after file entry but before test-specific args.